src/modules/game.module.ts

```diff
@@ -18,14 +18,14 @@ import { Game } from '../Hangman/ReadModels/game.entity';
   imports: [
     CqrsModule,
     EventStoreModule.registerFeature({
-      featureStreamName: '$ce-game',
+      featureStreamName: 'game',
       type: 'event-store',
       subscriptions: [
         {
           type: EventStoreSubscriptionType.CatchUp, // research various types
```

Comment:
check eventstore documentation. which subscription type to use to check the eventstore from a certain marker